dev/delve-debug

```diff
@@ -19,9 +19,9 @@ procname=${app/-}
 pod=$(kubectl get pod -n "$ns" -l "app=$app"  -o='jsonpath={.items[0].metadata.name}')
 
 kubectl --namespace "$ns" port-forward "pod/$pod" 4000 &
-trap "kill $!" EXIT
+trap 'kill $!' EXIT
 
-echo `tput bold`
+tput bold
 echo 'Please wait for the line "debug layer=debugger continuing" to appear...'
-echo `tput sgr0`
+tput sgr0
```

Comment:
Does that work? I was wondering about "echo tput", but didn't read up on it. I would have expected `echo -e $(tput ...)`